contracts/examples/StandardToken.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity ^0.4.24;
+
+import "../Initializable.sol";
+import "../token/ERC20/ERC20Detailed.sol";
+import "../token/ERC20/ERC20Mintable.sol";
+import "../token/ERC20/ERC20Pausable.sol";
+
+
+/**
+ * @title Standard ERC20 token, with minting and pause functionality.
+ *
+ */
+contract StandardToken is Initializable, ERC20Detailed, ERC20Mintable, ERC20Pausable {
+  function initialize(string name, string symbol, uint8 decimals, uint256 initialSupply, address initialHolder, address[] minters, address[] pausers) public initializer {
+    ERC20Detailed.initialize(name, symbol, decimals);
+
+    // Mint the initial supply
+    if (initialSupply > 0) { // To allow passing a null address when not doing any initial supply
+      _mint(initialHolder, initialSupply);
+    }
+
+    // Initialize the minter and pauser roles, and renounce them
+    ERC20Mintable.initialize(address(this));
+    renounceMinter();
```

Comment:
I'd avoid skipping the check altogether. We are precisely adding security checks on zOS to avoid uncalled initializers, and we would not be setting a good example by skipping them in the contracts we offer here. Let's leave it like this for now, this does not affect the end user interface, so we can change it later down the road if we find a better approach.